src/pages/profile/_id/index.vue

```diff
@@ -281,6 +281,12 @@ export default {
       }
       return '0'
     },
+    getTotalPlayTimeText(time) {
+      let hours = Math.floor(time / 36000)
```

Comment:
Just a random idiot passing by…
Aren't there 3600 seconds in an hour (and playtime is measured in seconds?)